pkg/csi/service/common/vsphereutil.go

```diff
@@ -250,7 +263,7 @@ func CreateBlockVolumeUtil(ctx context.Context, clusterFlavor cnstypes.CnsCluste
 // CreateFileVolumeUtil is the helper function to create CNS file volume with
 // datastores.
 func CreateFileVolumeUtil(ctx context.Context, clusterFlavor cnstypes.CnsClusterFlavor,
-	manager *Manager, spec *CreateVolumeSpec, datastores []*vsphere.DatastoreInfo) (string, string, error) {
+	manager *Manager, spec *CreateVolumeSpec, datastores []*vsphere.DatastoreInfo, cnsMgrSuspendCreateVolumeFSS bool) (string, string, error) {
```

Comment:
How about we rename the flag `cnsMgrSuspendCreateVolumeFSS` to filterSuspendedDatastores or something like that? Here and at other places. Should improve the readability IMO.